.github/workflows/rnmobile-android-runner.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         if: ${{ github.repository == 'WordPress/gutenberg' || github.event_name == 'pull_request' }}
         strategy:
             matrix:
-                native-test-name: [gutenberg-editor-initial-html]
+                native-test-name: [rendering]
```

Comment:
Using only `rendering` word might lead to executing other tests that contain this word. Hence, we could use `gutenberg-editor-rendering` instead, similar to what we had before. WDYT?

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/47814/commits/7c40a86360502ad8daf7a9ddb61431faefd0497a